include/detect_breakends/junction_detection.hpp

```diff
@@ -21,6 +21,9 @@
  *                                                           1: hierarchical_clustering,
  *                                                           2: self-balancing_binary_tree,
  *                                                           3: candidate_selection_based_on_voting)
+ * \param refinement_method method for refining breakends (0: no_refinement,
+ *                                                         1: sViper,
+ *                                                         2: prioritzation_based_on_rank_aggregation)
  *
  * \details Detects junctions from the CIGAR strings and supplementary alignment tags of read alignment records.
```

Comment:
I think that junctions and breakends represent places where the query sequence differs from the reference. But I'm not quite sure about the exact meaning of the two terms

Reply:
Thanks for the explanation! :)